### PR TITLE
Switch to XCode 11.7

### PR DESCRIPTION
**Pull request by cjh1**
pycifrw will not build with the latest version of XCode so downgrade to 11.7.

### CODE REVIEW

**Comment by cjh1** [comment]
@saransh13 This fixes #133. The problem was caused by [this upgrade](https://github.com/actions/virtual-environments/issues/1712). There is update upstream that fixes the package to build with the latest XCode, but its not in a release yet. I have requested the maintainer pushes a new release. For now this will get the CI back passing.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -31,6 +31,12 @@ jobs:
       run:
         shell: bash
     steps:
+    - name: Use XCode 12.0 so pycifrw will build
```
Comment (psavery) [inline .github/workflows/test.yml:34]: Should this be "11.7" instead of "12.0"?

Comment (cjh1) [reply .github/workflows/test.yml:34]: Indeed, I tried 12.0 first :-)

Comment (cjh1) [reply .github/workflows/test.yml]: Fixed

**Review by psavery — APPROVED** [review]
LGTM

**Comment by saransh13** [comment]
@psavery Thanks! Merging this.

**Pull request merged**